.travis.yml

```diff
@@ -79,7 +79,7 @@ matrix:
     - { env: ORCA_JOB=INTEGRATED_TEST_ON_PREVIOUS_MINOR, if: type = cron, name: "Integrated test on previous minor Drupal core version" }
     - { env: ORCA_JOB=INTEGRATED_UPGRADE_TEST_FROM_PREVIOUS_MINOR, if: type = cron, name: "Integrated upgrade test from previous minor" }
     - { env: ORCA_JOB=ISOLATED_TEST_ON_CURRENT ACMS_JOB=base_full, if: type = cron, name: "Isolated test on current Drupal core version" }
-    - { env: ORCA_JOB=ISOLATED_TEST_ON_CURRENT, if: type = cron, php: "8.0", dist: focal, name: "Isolated test on current Drupal core version and PHP 8" }
```

Comment:
@panshulK My bad. I think let keep this job as well. Because we currently support PHP 8.0 as well.